code/WorkInProgress/TempEngine.dm

```diff
@@ -559,7 +700,19 @@
 	*/
 */
 	on_burn()
-		var/additional_heat = src.fuel * 4
+
+		// -(1.2x - 1)^2 + 1 expands to 2.4x-1.44x^2
+		// -0.48x*(3x-5)
+		//)
```

Comment:
?

Reply:
Showing work for curve I wanted to follow.  Intended as rationale for magic number.

Reply:
The last line

Reply:
Oh... well... fuck.